var-core/src/main/java/io/varhttp/VarWebSocketMatcher.java

```diff
@@ -0,0 +1,23 @@
+package io.varhttp;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Optional;
+
+public class VarWebSocketMatcher implements ControllerMatcher {
+	private final RegisteredWebSockets registeredWebSockets;
+
+	public VarWebSocketMatcher(RegisteredWebSockets registeredWebSockets) {
+		this.registeredWebSockets = registeredWebSockets;
+	}
+
+	@Override
+	public Optional<ControllerMatch> find(Method method) {
+		WebSocket controllerAnnotation = method.getAnnotation(WebSocket.class);
```

Comment:
What about `registeredWebSockets`, should we also use it for filtering out?

Reply:
removed